### ISSUE TITLE
double instead of single-quotes in log snippets

### ISSUE THREAD

**Issue by MartinP7r** [onecentlin/laravel5-snippets-vscode#28]
Since I most often have to delete the single quotes in the log snippets in order to use string interpolation like 
```php
\Log::debug("some $variable here");
```

I'd propose to either replace all single quoted log methods in https://github.com/onecentlin/laravel5-snippets-vscode/blob/master/snippets/log.json with double-quotes or make the quotes part of the initial tabbable shortcuts like

```json
"Log::debug(${1:'${2:message}'});$3"
```

I could submit a PR if you'd like. If there's no preference I'd go with the first solution.

**Comment by MartinP7r** [comment]
Any thoughts on this?

**Comment by onecentlin** [comment]
@MartinP7r  Sure, please summit a PR. Thanks.

**Comment by MartinP7r** [comment]
Merged in #32
